### PR TITLE
[WIP] Reorder so flake8 comes first before builds

### PR DESCRIPTION
**Pull request by mike-wendt**
This way we don't have to wait 30 mins for a failure on styling.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,19 +20,22 @@ matrix:
...
   - export PATH=$HOME/miniconda3/bin:$PATH
-
-install:
   # check
   - conda --version
   # install conda build
   - conda install conda-build anaconda-client flake8 conda-verify cmake=3.12 --yes
+
+install:
+  # check flak8 before proceeding
```
Comment (kkraus14) [inline .travis.yml]: Typo here

**Comment by kkraus14** [comment]
I understand wanting flake failures before build / testing but this does prevent people from getting development results unless their code passes style checks, and I know some developers only worry about styling once the code is working as expected.

**Comment by harrism** [comment]
That's a good point. Since flake8 failures don't have to break the build, is it possible to show the flake8 failures, proceed with the next steps, and then make the overall result pass only if everything passed?

**Comment by kkraus14** [comment]
@mike-wendt We'd have to do some bash hackery of saving the exit codes of the incremental commands and do an AND at the end to see if the build passes, what are your thoughts?

File: .travis.yml (hunk 1 above)
Comment (kkraus14) [inline .travis.yml]:
```suggestion
  # check flake8 before proceeding
```

**Comment by mike-wendt** [comment]
Closing this PR as this will be resolved with the new CI approach

**Pull request closed without merging**